### ISSUE TITLE
Symfony 4.3 EventDispatcher deprecation

### ISSUE THREAD

**Issue by enumag** [overblog/GraphQLBundle#493]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Version/Branch   | 0.11.12

```
User Deprecated: Calling the "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" method with the event name as first argument is deprecated since Symfony 4.3, pass it second and provide the event object first instead.
```

The easy fix is of course swapping the arguments and raising the version constraint for symfony/event-dispatcher.

If you want compatibility with older Symfony as well then it gets a bit more complicated but doable.

**Comment by mcg-web** [comment]
Since we want compatibility with older Symfony indeed it will be a little more complicated. I'll try to fix this when making the bundle supports Symfony 4.3 in the coming days.

**Comment by mcg-web** [comment]
Fixed and released in [v0.11.13](https://github.com/overblog/GraphQLBundle/releases/tag/v0.11.13). Thank you for reporting this :+1: .
